src/main/java/cam72cam/immersiverailroading/entity/EntityMoveableRollingStock.java

```diff
@@ -344,6 +349,22 @@ public void onTick() {
                         wheel_sound.stop();
                     }
                 }
+                if(MinecraftClient.getPlayer().getPosition().distanceTo(getPosition()) < 40){
```

Comment:
This completely breaks the previous code block.  How is flange different than wheel sound?

Reply:
copy&paste error